### PR TITLE
docs: move RFC_0120 to stable

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
This moved RFC_0120 to stable. 
Sort maintainers list by alphabetical order.

### CODE REVIEW

**Comment by CjS77** [comment]
Add a `Change Log` please (and fill it in)

**Comment by CjS77** [comment]
* Broken link in line #85
* Burns are stable now, and there's no mention of them here

**Comment by CjS77** [comment]
Fix the link in L#86 and the suggestion should fix the LaTeX error, and we should be good here.

**Comment by CjS77** [comment]
pneding changes..
ACK

**Comment by stringhandler** [comment]
The transaction input section is missing Tariscript fields and the maturity check seems wrong to me

**Comment by stringhandler** [comment]
I would also say that there is a set of features that are allowed and a number of versions that are allowed in blocks, inputs, outputs and features. Tariscript also has a restriction on the version and opcodes allowed

**Comment by stringhandler** [comment]
There is no mention of the Tariscript balance checks done

**Comment by stringhandler** [comment]
Where hashing is mentioned, the domain for the hasher should also be specified

**Comment by stringhandler** [comment]
`Total difficulty` is no longer part of the header and should be removed

**Comment by stringhandler** [comment]
The script offset is referred to as `total_script_offset` in one place and `script_offset` in another

**Comment by stringhandler** [comment]
The section `Total accumulated proof of work` should mention that this is a u128 to allow for the product of two u64 numbers

**Comment by stringhandler** [comment]
Add `kernels are sorted by excess_sig` to `Transaction Ordering` section

**Comment by stringhandler** [comment]
"Any time that is more than the FTL is rejected until such a time that it is not ~~less~~ more than the FTL."

**Comment by stringhandler** [comment]
Closes #43

File: src/RFC-0120_Consensus.md [hunk 1]
```diff
@@ -359,16 +356,25 @@ final Merkle root. Input, output, and kernel ordering within a block is, therefo
 The block MUST be transmitted in canonical ordering. The advantage of this approach is that sorting does not need to be 
 done by the whole network, and verification of sorting is exceptionally cheap.
 
-Transaction outputs are sorted lexicographically by the byte representation of their Pedersen commitment i.e. `\\(k \cdot G + v \cdot H\\)`.
+Transaction outputs are sorted lexicographically by the byte representation of their Pedersen commitment i.e. ( \\(k \cdot G + v \cdot H\\) ).
 Transaction inputs are sorted lexicographically by the [hash of the output](RFC-0121_ConsensusEncoding.md#transaction-output) that is spent by the input.
```
Comment (stringhandler) [inline src/RFC-0120_Consensus.md:360]: Transaction kernels are sorted lexicographically by the excess signature byte representation.

**Review by stringhandler — APPROVED** [review]
just one more

File: src/RFC-0120_Consensus.md [hunk 2]
```diff
@@ -359,16 +356,25 @@ final Merkle root. Input, output, and kernel ordering within a block is, therefo
 The block MUST be transmitted in canonical ordering. The advantage of this approach is that sorting does not need to be 
 done by the whole network, and verification of sorting is exceptionally cheap.
 
-Transaction outputs are sorted lexicographically by the byte representation of their Pedersen commitment i.e. `\\(k \cdot G + v \cdot H\\)`.
+Transaction outputs are sorted lexicographically by the byte representation of their Pedersen commitment i.e. ( \\(k \cdot G + v \cdot H\\) ).
```
Comment (CjS77) [inline src/RFC-0120_Consensus.md:359]:
```suggestion
Transaction outputs are sorted lexicographically by the byte representation of their Pedersen commitment i.e. ( \\(k \cdot G + v \cdot H\\) ).

Transaction kernels are sorted lexicographically by the excess signature byte representation.
```

**Review by CjS77 — APPROVED** [review]
LGTM

**Pull request merged**